requirements.txt

```diff
@@ -2,6 +2,7 @@
 
 click>=8.1.7
 dictdiffer>=0.9.0
+gitpython>=3.1.41
```

Comment:
Please add this to the internal pypi so if there is any issue with this we will know right away.

Reply:
I started using it after checking that its already there https://github.com/getsentry/pypi/blob/main/packages.ini#L497